Fix infinite recursion on Darwin.

matthewbauer opened the pull request:
Fixes #18

matthewbauer committed:
Fix infinite recursion on Darwin.

Fixes #18

ElvishJerricco commented:
@matthewbauer Whoa this is all it took? Does most of release.nix build with this change?

matthewbauer commented:
It gets things to evaluate. Still waiting on LLVM to build, but I expect that it won't need libcxxabi. I think that was put there for some bootstrapping reasons that I can't remember now.

matthewbauer committed:
Set install name correctly for LLVM.

This was already fixed upstream in Nixpkgs, but not fixed here.

https://github.com/NixOS/nixpkgs/commit/adcf4aa5241f8dc37195d9bcbce27a80b913a512

matthewbauer commented:
This works now with these commits + this commit from Nixpkgs:

https://github.com/NixOS/nixpkgs/commit/bf041c3f1d5e63e27b531faf69244fe99fcfd6c1
